tests/lib/managers/terms-of-service-test.js

```diff
@@ -0,0 +1,571 @@
+			var response = {
+				type: 'terms_of_service_user_status',
+				id: '1234',
+				tos: {},
+				user: {},
+				is_accepted: true,
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('put').withArgs('/terms_of_service_user_statuses/1234', expectedParam)
+				.returns(Promise.resolve(response));
+			termsOfService.updateUserStatuses(TERMS_OF_SERVICE_USER_STATUS_ID, true);
+		});
+
+		it('should call callback with updated terms of service user status info when a 200 response is returned', function(done) {
+			var response = {
+				type: 'terms_of_service_user_status',
+				id: '1234',
+				tos: {},
+				user: {},
+				is_accepted: true,
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'put').yieldsAsync(null, response);
+			termsOfService.updateUserStatuses(TERMS_OF_SERVICE_USER_STATUS_ID, true, function(err, termsOfServiceUserStatusObject) {
+				assert.ifError(err);
+				assert.strictEqual(termsOfServiceUserStatusObject, response, 'terms of service user object is returned');
+				done();
+			});
+		});
+
+		it('should return a promise resolving to updated terms of service user status object when a 200 response is returned', function() {
+			var response = {
+				type: 'terms_of_service_user_status',
+				id: '1234',
+				tos: {},
+				user: {},
+				is_accepted: true,
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'put').returns(Promise.resolve(response));
+			return termsOfService.updateUserStatuses(TERMS_OF_SERVICE_USER_STATUS_ID, true)
+				.then(termsOfServiceUserStatusObject => {
+					assert.strictEqual(termsOfServiceUserStatusObject, response, 'terms of service user status object is returned');
+				});
+		});
+	});
+
+	describe('setUserStatuses()', function() {
+		var expectedParam = {body: {tos: {tos_id: '1234', type: 'terms_of_service'}, is_accepted: true}};
+		it('should make POST request to create custom terms of service user status when called', function() {
+			var response = {
+				statusCode: 200,
+				body: {
+					type: 'terms_of_service_user_status',
+					id: '1234',
+					tos: {},
+					user: {},
+					is_accepted: true,
+					created_at: '2017-10-21T20:09:59-07:00',
+					modified_at: '2017-10-21T20:09:59-07:00'
+				}
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('post').withArgs('/terms_of_service_user_statuses', expectedParam)
+				.returns(Promise.resolve(response));
+			termsOfService.setUserStatuses(TERMS_OF_SERVICE_ID, true, TERMS_OF_SERVICE_USER_ID);
```

Comment:
You're setting a user ID here but not verifying it anywhere in the test — need to make sure it's being correctly used